presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/common/hadoop/supervisorctl-status.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
This file is added by `Interrupt tests on unhealthy environment` commit but not used here.

Reply:
Also it is removed in the next commit